doc/en/user/source/community/vector-mosaic/delegate.rst

```diff
@@ -61,3 +61,30 @@ in a single map:
 .. figure:: images/places-mosaic.png
    :align: center
 
+Creating A FlatGeobuf Delegate and Granules with ogr2ogr and ogrtindex
+======================================================================
+
+`FlatGeobuf <https://flatgeobuf.org>`_ files are an excellent option for cloud storage of granule data due the built in support for R-Tree indices and the use of `HTTP Range requests <https://developer.mozilla.org/en-US/docs/Web/HTTP/Range_requests>`_ to limit the amount of data streamed over the network.
+
+``ogr2ogr`` can be used to convert a directory of shapefiles into a directory of indexed FlatGeobufs using a simple bash script like below
+
+.. code-block:: bash
+
+   #!/bin/bash
+   shopt -s nullglob
+   FILES="/data/tiger/*.shp"
+   for f in $FILES
+   do
+     fgbfilename="$(basename $f .shp).fgb"
+     ogr2ogr -f FlatGeobuf $fgbfilename $f -nlt PROMOTE_TO_MULTI -lco SPATIAL_INDEX=YES
+   done
+
+Here is an example that generates a delegate FlatGeobuf from the directory of FlatGeobufs.  The third step below uses ``ogr2ogr`` commandline to trim a comma and number that ``ogrtindex`` appends to the end of the granule reference, and to turn the file location into a valid URL.  Note that we use the ``*.fgb`` extension wildcard instead of ``*.shp`` that was used in the previous example. Also note the exclusion of the ``write_absolute_path``.  Instead we append the AWS S3 bucket URL to the generated filename.  
+
+#. Switch to directory with the FlatGeoBufs.
+#. ogrtindex -tileindex "params" delegate_raw.shp \*.fgb
+#. ogr2ogr delegate.shp delegate_raw.shp -dialect SQLite -sql "SELECT Geometry,'https://mybucketname.s3.amazonaws.com/'||SUBSTR(params,1,LENGTH(params)-2) AS params from delegate_raw"
+#. ogr2ogr -f FlatGeobuf delegate.fgb delegate.shp -nlt PROMOTE_TO_MULTI -lco SPATIAL_INDEX=YES
```

Comment:
Rather than turning the delegate to flatgeobuf, I would keep it in shp and then suggest to turn it into a postgis table instead, useful if someone needs to add and remove new granules over time (it would support concurrent edits, high rate loading, and so on).